composer.json

```diff
@@ -109,30 +108,32 @@
         ]
     },
     "require": {
-        "php": ">=8.0",
+        "php": ">=8.2",
         "composer/installers": "^1.7",
         "cweagans/composer-patches": "^1.7",
         "drupal/admin_denied": "^2",
-        "drupal/components": "^2.4",
+        "drupal/classy": "^1.0",
```

Comment:
Do we need classy specifically? Our base-theme currently uses [contrib stable](https://www.drupal.org/project/stable) as its base. The diff shows we're also adding `stable:^2.0` later on in the composer file.

If another active theme such as Claro needs it, then ignore me.

Reply:
I was prompted to add it in https://github.com/UN-OCHA/common-design-site/pull/329/commits/eedb60b8dee633fbea7ab67ae9ca3e36ce79f9d9, but didn't make notes as to where the prompt came from. 

It's not required by Claro: https://www.drupal.org/project/drupal/issues/3115088

I've made a ticket to investigate further and see if we can remove it again: https://humanitarian.atlassian.net/browse/CD-512

Reply:
I see Claro uses Classy as its base theme https://git.drupalcode.org/project/claro/-/blob/8.x-2.x/claro.info.yml?ref_type=heads#L15 and it doesn't seem to list Classy as a dependency so I would wonder if we don't include it here, we will get composer install issues?
I also see Classy included in the composer.json for RW, SLT, ODSG I assume for the same reason.

Reply:
Correction https://git.drupalcode.org/project/drupal/-/blob/11.x/core/themes/claro/claro.info.yml?ref_type=heads#L15
Since Claro was moved to Core, I think Classy is no longer a dependency. If removing it is possible, let's do so.